src/test/java/com/williamfeliciano/sba_movies/services/MoviesServiceTests.java

```diff
@@ -0,0 +1,81 @@
+package com.williamfeliciano.sba_movies.services;
+
+import com.williamfeliciano.sba_movies.dtos.MovieDetailsDto;
+import com.williamfeliciano.sba_movies.dtos.MovieDto;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
For the moment, you don't need this annotation as you aren't using any mock. MoviesService has no dependencies injected, no need to mock.

You could write the entire MoviesServiceTests with using Mockito. But leave it that way, you will be ready when you have the mappers or repositories.